payload/run.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# required env vars:
+# $BOT_PAT
+# $WHEEL_RELEASE_ARTIFACTS
+# optional:
+# $VLLM_TGIS_ADAPTER_VERSION
+# $VLLM_WHEEL_VERSION
+set -ex
+
+cat <<EOF > ${HOME}/.netrc
+machine gitlab.com
+login rhel-ai-wheels-prefetch-token-rhoai 
+password $BOT_PAT
+EOF
+
+trap "rm ${HOME}/.netrc" EXIT
+
+# https://docs.astral.sh/uv/configuration/indexes/#searching-across-multiple-indexes
+# This will prefer to use the custom index, and fall back to pypi if needed
+export UV_EXTRA_INDEX_URL=${VLLM_WHEEL_INDEX}
```

Comment:
This won't work, RHEL AI wheels are an all-or-nothing solution which cannot be mixed with PyPi wheels